src/odin/enhancedtrippath.cc

```diff
@@ -523,19 +523,21 @@ std::vector<std::pair<std::string, bool>> EnhancedTripLeg_Edge::GetNameList() co
   return name_list;
 }
 
-std::string EnhancedTripLeg_Edge::GetLevelRef() const {
-  std::string level_ref;
+std::vector<std::string> EnhancedTripLeg_Edge::GetLevelRef() const {
```

Comment:
When there are no `level_ref` values, fall back to the `level` ones.

Reply:
Edit: only if there is a single level value though; when forming the text instructions and a maneuver ends on a flight of stairs, we don't want the instruction to be "Take the stairs to level 1;2".